src/components/global/Playground/index.tsx

```diff
@@ -129,11 +165,42 @@ export default function Playground({
     const codeSnippets = {};
     Object.keys(code).forEach((key) => {
       // Instantiates the React component from the MDX content.
-      codeSnippets[key] = code[key]({});
+      if (typeof code[key] === 'function') {
+        codeSnippets[key] = code[key]({});
+      } else if (typeof code[key] === 'object') {
+        const fileSnippets = {};
+        for (const fileName of Object.keys(code[key].files)) {
+          fileSnippets[`${fileName}`] = code[key].files[fileName]({});
+        }
+        codeSnippets[key] = fileSnippets;
+      }
     });
     setCodeSnippets(codeSnippets);
   }, []);
 
+  function getCodeSnippetId(fileName: string) {
+    let fileNameId = fileName;
+    // replace all non-alphanumeric characters with underscores
+    fileNameId = fileNameId.replace(/[^a-zA-Z0-9]/g, '_');
+    return `playground_${id}_${fileNameId}`;
+  }
+
+  function renderCodeSnippets() {
+    if (codeSnippets[usageTarget]) {
+      if (typeof codeSnippets[usageTarget].$$typeof === 'symbol') {
```

Comment:
What does this line do?

Reply:
It is the uglier solution for knowing if the object is MDX content vs. the object syntax, I will flip this to the updated syntax from Amanda's feedback and add a comment explaining it.